addons/binding/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/PowerControlPower.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lgwebos.internal;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.lgwebos.handler.LGWebOSHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.connectsdk.device.ConnectableDevice;
+import com.connectsdk.service.capability.PowerControl;
+
+/**
+ * Handles Power Control Command.
+ * Note: Connect SDK only supports powering OFF for most devices.
+ *
+ * @author Sebastian Prehn
+ * @since 1.8.0
+ */
+public class PowerControlPower extends BaseChannelHandler<Void> {
+    private Logger logger = LoggerFactory.getLogger(PowerControlPower.class);
```

Comment:
Imho its better to have the logger's final, I hope you will never change them and imho it creates less noise then adding them to parameters.

Reply:
Done 10x